src/helpers/eventBridge.ts

```diff
@@ -122,12 +128,12 @@ export default class EventBridge {
       })
       .promise();
 
-    await this.getEvents(); // need to clear this manual published event from the SQS observer queue.
+    await this.getEvent(); // need to clear this manual published event from the SQS observer queue.
```

Comment:
This is interesting behaviour, perhaps the documentation should be updated to reflect that calling `getEvent` or `getEvents` will also clear the queue

Reply:
Thanks for your comments, I took them into account !